src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -1047,7 +1047,7 @@ private String getPermissionFilterQuery(DataverseRequest dataverseRequest, SolrQ
         // add joins on all the non-public groups that may exist for the
         // user:
 
-        // Authenticated users and GuestUser may be part of one or more groups; such
+        // Authenticated users - and GuestUser - may be part of one or more groups; such
```

Comment:
```suggestion
        // Authenticated users, and GuestUser, may be part of one or more groups; such
```

Or maybe "(and GuestUser)"? The dashes throw me off a bit.

Reply:
Should have mentioned - this was in response to https://github.com/IQSS/dataverse/pull/10706#discussion_r1815453266 indicating that it's useful to emphasize that the GuestUser can also be in groups. I tried a switch to using * instead.

Reply:
Looks great now. Very clear. Thanks. Approved.